style.css

```diff
@@ -966,40 +972,40 @@ label {
   .tonicc {
     margin-left: 46px;
   }
-  
+
   .container {
     display: flex;
   }
-  
+
   .text-container {
     display: flex;
     align-items: center;
     margin-top: 20px;
   }
-  
+
   .cano {
     font-size: 13px;
     color: #344563;
     font-weight: 700;
   }
-  
+
   .nopi {
     font-size: 13px;
     margin-left: 12px;
     color: #7a869a;
     font-weight: 600;
   }
-  
+
   .text-container > img {
     width: 8px;
     height: 8px;
   }
-  
+
   .languages {
     display: flex;
     margin-top: 28px;
   }
-  
+
   .my-button {
     display: flex;
     flex-direction: row;
```

Comment:
- [ ] To enhance the interactivity and improve the user experience, kindly consider adding a `hover` effect to your `button`, as shown in the `Figma` design. Adding a `hover` effect can provide visual feedback to users, making the `button` more interactive and engaging. See below for more details:
<img width="752" alt="Screenshot 2023-06-15 at 11 53 21" src="https://github.com/santitoot/desktop/assets/107192076/31841df3-2d1d-4d77-90a7-f63aeeb87a2b">